operator/v1/types_network.go

```diff
@@ -111,6 +111,10 @@ type AdditionalNetworkDefinition struct {
 	// rawCNIConfig is the raw CNI configuration json to create in the
 	// NetworkAttachmentDefinition CRD
 	RawCNIConfig string `json:"rawCNIConfig"`
+
+	// sriovConfig configures the openshiftSRIOV plugin.
+	// +optional
+	SRIOVConfig *SRIOVConfig `json:"sriovConfig,omitempty"`
```

Comment:
should mark `RawCNIConfig` as `omitempty` now too

Reply:
Thanks Dan! I noticed that this has been changed in Tomo's [PR](https://github.com/openshift/api/pull/257), so I'll keep as it is for this PR, does it sound ok?